src/tox/session/commands/run/parallel.py

```diff
@@ -25,7 +26,7 @@ def run_parallel(config, venv_dict):
     semaphore = Semaphore(max_parallel)
     finished = Event()
 
-    show_progress = not live_out and reporter.verbosity() > reporter.Verbosity.QUIET
+    show_progress = (not live_out or not nospinner) and reporter.verbosity() > reporter.Verbosity.QUIET
```

Comment:
I think it'd cleaner this way:
```suggestion
    show_progress = not nospinner and not live_out and reporter.verbosity() > reporter.Verbosity.QUIET
```
plus it'd abort other checks immediatelly...